scripts/jest/shouldIgnoreConsoleError.js

```diff
@@ -32,15 +32,6 @@ module.exports = function shouldIgnoreConsoleError(
         // We haven't finished migrating our tests to use createRoot.
         return true;
       }
-      if (
-        TODO_ignoreHydrationErrors &&
```

Comment:
Me: wtf is TODO_ignoreHydrationErrors, kill that

*checks blame*

Me: oh yeah great idea, very smary